config/sync/system_status.settings.yml

```diff
@@ -0,0 +1,3 @@
+system_status_token: 55348469994162l
+system_status_encrypt_token: D79hL939883
```

Comment:
`encrypt_token` sounds like something that should not be in git. Can a 3rd party impersonate calls on our behalf with these tokens?

Reply:
If we deploy this to both `stg` and `prod` can the lumtrio service distinguish between the two?

Reply:
The UUID is a combination of these tokens, I have no idea on how this works

Reply:
If you search for `system_status_token` under the `aakb` org on github you can see how we do on other sites like: https://github.com/aakb/cfia/blob/d47f841dc11ec87514f26dcda974b957d32dfbc9/config/sync/system_status.settings.yml

Reply:
This needs to be moved to `settings.local.php`